services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -1841,6 +1841,24 @@ module.exports = {
       return c;
     },
 
+    /**
+     *
+     */
+    contentCompanyField: async (_, { input }, { base4rest, basedb }, info) => {
+      validateRest(base4rest);
+      const company = 'platform/content/company';
+      const { id, companyId } = input;
+      const content = await basedb.strictFindById('platform.Content', id, { projection: { type: 1 } });
+      const model = `platform/content/${dasherize(content.type)}`;
+      const body = new Base4RestPayload({ type: model });
+      if (companyId) body.setLink('company', { id: companyId, type: company });
+      else body.unsetLink('company');
```

Comment:
Not sure why the linter is missing this, but fix your if/else clause: https://github.com/airbnb/javascript#blocks

Reply:
Bonus points for fixing the linter :)